Source/WebCore/page/scrolling/ScrollingTree.cpp

```diff
@@ -423,9 +427,11 @@ bool ScrollingTree::commitTreeStateInternal(std::unique_ptr<ScrollingStateTree>&
 
     for (auto nodeID : commitState.unvisitedNodes) {
         LOG_WITH_STREAM(Scrolling, stream << "ScrollingTree::commitTreeState - removing unvisited node " << nodeID);
-        removeNode(nodeID);
+        removeNode(nodeID, commitState.frameHostingNode.get());
     }
 
+    LOG_WITH_STREAM(ScrollingTree, stream << "committed ScrollingTree" << scrollingTreeAsText(debugScrollingStateTreeAsTextBehaviors));
```

Comment:
Is there a reason this log was moved?

Reply:
I added this before the recursive commits below as it made debugging confusing when the scrolling trees would be printed out of order.